src/main/java/mod/gate/core/mixin/GameJoinMixin.java

```diff
@@ -0,0 +1,18 @@
+package mod.gate.core.mixin;
+
+import mod.gate.core.events.EventHandler;
+import mod.gate.core.events.JoinWorldEvent;
+import net.minecraft.client.network.ClientPlayNetworkHandler;
+import net.minecraft.network.packet.s2c.play.GameJoinS2CPacket;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+@Mixin(ClientPlayNetworkHandler.class)
+public class GameJoinMixin {
```

Comment:
The event is called JoinWorldEvent, What are we going with? MC terminology or a standard we can agree on